ts/components/messages/MessagesInbox.tsx

```diff
@@ -74,6 +78,14 @@ const generateMessagesStateNotArchivedArray = (
     []
   );
 
+const ListEmptyComponent = (
+  <View style={styles.emptyListWrapper}>
+    <Image
+      source={require("../../../img/messages/empty-message-list-icon.png")}
```

Comment:
why we have both the png and the svg? can't we just use the svg?

Reply:
Currently we cannot use svg without using further libraries (like for example [react-native-svg-uri](https://github.com/vault-development/react-native-svg-uri)), would you prefer me to remove the svg file from this branch at the moment?